og_ui/og_ui.test

```diff
@@ -291,6 +291,13 @@ class OgUiManagePeopleTestCase extends BackdropWebTestCase {
     $user1 = $this->backdropCreateUser();
     $this->backdropLogin($user1);
 
+    // The field og_user_og_entity_test was added automatically to the user 
+    // entities by og_field_create_instance() when field 
+    // group_group was attached to og_entity_test in setUp().
+    // It must be deleted in order for upcoming tests to pass.
+    // See the equivalente line in the most recent D7 og_ui.test.
```

Comment:
Spelling mistake: `equivalente` should be `equivalent`.

Reply:
@argiepiano Looks great, the test is green in the checks:

> Organic groups UI: OG UI manage people (OgUiManagePeopleTestCase) 126 passes, 0 fails, 0 exceptions, and 38 debug messages. [17.927s]

Reply:
I just fixed the typo!